godtools/App/Features/AppLanguage/Domain/UseCases/SearchAppLanguageInAppLanguagesListUseCase.swift

```diff
@@ -17,7 +17,7 @@ class SearchAppLanguageInAppLanguagesListUseCase {
         self.getAppLanguagesListUseCase = getAppLanguagesListUseCase
     }
     
-    func getSearchResultsPublisher(for searchTextPublisher: CurrentValueSubject<String, Never>) -> AnyPublisher<[AppLanguageListItemDomainModel], Never> {
+    func getSearchResultsPublisher(for searchTextPublisher: AnyPublisher<String, Never>) -> AnyPublisher<[AppLanguageListItemDomainModel], Never> {
         
```

Comment:
Hey @rachaelblue I may come back to this later and see about moving getAppLanguagesListUseCase.getAppLanguagesListPublisher() out of this UseCase and instead injecting the list of languages as a publisher in the method call along with the searchTextPublisher.  The 1 advantage there is it would make writing tests a little be easier.  For now though can leave as is.